test/unit_tests/test_simulator.py

```diff
@@ -16,11 +16,18 @@ def test_add_ocean_floor(self):
 
         simulator = Simulator(4000, gui=False)
 
+        temp_load = PybulletAPI.loadURDF
+        temp_change_visual_shape = PybulletAPI.changeVisualShape
+
         PybulletAPI.loadURDF = Mock()
         PybulletAPI.changeVisualShape = Mock()
 
         simulator.add_ocean_floor(100)
 
 
```

Comment:
Yes there is a better way. Use Mock.Patch() 
Example usage here: https://github.com/LOBSTER-Robotics/lobster-auv/blob/6de1edad5c42c8b7f2188dbde82705a84b518d27/test/integration/simulation_tests/test_depth_integrated.py
```suggestion
        with mock.patch("lobster_simulator.common.pybullet_api.PybulletAPI.loadURDF") as loadURDFMock:
            with mock.patch("lobster_simulator.common.pybullet_api.PybulletAPI.changeVisualShape") as changeVisualShapeMock:
                simulator.add_ocean_floor(100)


```

Reply:
After applying suggestion, line 32 and 33 can be removed. Also lines 28-29 have to be incremented

Reply:
You could also replace the 'with' lines with a decorator

Reply:
Ah yes that is indeed a better solution

Reply:
I have updated it